GMnetENGINE.gmx/scripts/htme_shutdown.gml

```diff
@@ -33,6 +33,8 @@ for(var i=0; i<ds_map_size(self.localInstances); i+=1) {
         if wasServer 
             htme_serverRemoveBackup(key);
         with inst_id {instance_destroy();}
+    } else if wasServer {

```

Comment:
The server must remove backups if the server disconnects and the other players is in another room. Why didn't I see this earlier?